### PR TITLE
Added embeds for all basic bot commands

### PR DESCRIPTION
**Pull request by sdk2k01**
@Dsantra92 
all basic bot commands and error messages give errors in vibrant embeds.

<details>
<summary>Image showing new vibrant embeds for bot command outputs and error messages.</summary>
Screenshot 2021-08-14 at 11 04 30 PM
</details>

### CODE REVIEW

File: src/cogs/basic.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from discord.ext import commands, tasks
 from itertools import cycle
 
-status = cycle(['arXiv','Kaggle'])
+status = cycle(['arXiv','Kaggle','redditAPI'])
```
Comment (Dsantra92) [inline src/cogs/basic.py:7]: status = cycle([ 'arXiv', 'Kaggle', 'redditAPI' ])

File: src/cogs/basic.py [hunk 2]
```diff
@@ -25,7 +25,11 @@ async def on_command_error(self, ctx, error):
 
     @commands.command()
     async def ping(self, ctx):
-        await ctx.send(f'Pong! In {round(self.client.latency*1000)} ms.')
+        embed=discord.Embed(title="Ping",
```
Comment (Dsantra92) [inline src/cogs/basic.py:24]: Give a space before and after `=`

File: src/cogs/basic.py [hunk 3]
```diff
@@ -17,32 +18,42 @@ async def on_ready(self):
...
+        color=discord.Color.green())
+        embed.set_footer(text=f'Information requested by : {ctx.author.display_name}')
+        await ctx.send(embed=embed)
 
     @commands.command()
     @commands.has_permissions(manage_messages=True)
     async def clear(self, ctx, amount : int):
         await ctx.channel.purge(limit=amount)
+        embed=discord.Embed(title="Purged Messages!",description=f'{amount} messages have been cleared.',
+        color=discord.Color.teal())
```
Comment (Dsantra92) [inline src/cogs/basic.py:35]: spacing issue with `=` and `,`

File: src/cogs/basic.py [hunk 4]
```diff
@@ -17,32 +18,42 @@ async def on_ready(self):
...
+        await ctx.channel.purge(limit=1)
 
     @clear.error
     async def on_command_err(self, ctx, error):
         if isinstance(error, commands.MissingRequiredArgument):
-            await ctx.send('Please specify an amount of messages to be deleted!')
+            embed_title="Missing Required Argument!"
+            embed_desc="Please specify an amount of messages to be deleted!"
+            embed_color=discord.Color.orange()
+            #await ctx.send('Please specify an amount of messages to be deleted!')
```
Comment (Dsantra92) [inline src/cogs/basic.py:46]: spacing issue `=`

**Review by Dsantra92 — CHANGES_REQUESTED** [review]
There are some minor changes required for maintaining style consistency in the codebase. I have suggested changes only in some of the lines, but the changes are to be made for all lines with similar styling problems.

**Comment by sdk2k01** [comment]
Updated PR.
The changes requested have been made. Space discrepancies are removed and comments are added.
<details>
<summary>A brief preview of the changes</summary>
Screenshot 2021-08-15 at 10 03 50 AM
</details>

**Comment by Dsantra92** [comment]
Bhery Kull deer!!

**Pull request merged**